exo/inference/mlx/sharded_utils.py

```diff
@@ -184,7 +184,7 @@ async def load_shard(
     processor.encode = processor.tokenizer.encode
     return model, processor
   else:
-    tokenizer = await resolve_tokenizer(model_path)
```

Comment:
Why did you change this?

Reply:
Not a good reason to change it. Please change it back to `resolve_tokenizer`. If we run into issues with that, we should fix those issues, not revert back to the old version